### ISSUE TITLE
Heading rows and columns might be missing markup in the view.

### ISSUE THREAD

**Issue by jodator** [ckeditor/ckeditor5-table#18]
The heading rows and heading columns cells[ should have "scope" attribute](https://developer.mozilla.org/en-US/docs/Learn/HTML/Tables/Advanced#Tables_for_visually_impaired_users) set for accessibility reasons:

> Screenreaders will recognize markup structured like this, and allow their users to read out the entire column or row at once, for example.

Example proper markup of a table with one heading 

```html
<table>
	<thead>
		<tr>
			<th scope="col">a</th>
			<th scope="col">b</th>
		</tr>
	</thead>
	<tbody>
		<tr>
			<th scope="row">c</th>
			<td>d</td>
		</tr>
	</tbody>
</table>
```

**Comment by dtwist** [comment]
Since you're discussing markup: I posted these styles in #1 earlier as an exploration of @oleq's default table design:
https://codepen.io/david-twist/pen/wjyGpo

In addition to scope attributes on cells, to get the vertical rule between the last column `<th>` elements, you also need `<colgroup>` elements with a class to differentiate between data and header columns _[note: lone `<col>` elements have quirky support for classes, so instead always use `<colgroup>`, which are implied by a `<col>` anyhow ]_:

```html
<table class="ck-table">
  <caption>Single Column and Row Headers</caption>
  <colgroup class="ck-table_head-col"></colgroup>
  <colgroup class="ck-table_data-col" span="2"></colgroup>
  <thead>
    <tr>
      <td>&nbsp;</td>
      <th scope="col">Hot</th>
      <th scope="col">Cold</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <th scope="row">Fire</th>
      <td>Yes</td>
      <td>&nbsp;</td>
    </tr>
    <tr>
      <th scope="row">Ice</th>
      <td>&nbsp;</td>
      <td>Yes</td>
    </tr>
  </tbody>
</table>
```

I mention this because it is tangentially related, but—irrespective of using them as styling hooks—colgroups are important for A11y. See WAI docs here:
https://www.w3.org/WAI/tutorials/tables/

You also want to take care to create multiple `<colgroup>`s and set `scope="colgroup"` when you have any cells with `colspan` > 1:

```html
<table class="ck-table">
  <colgroup class="ck-table_head-col"></colgroup>
  <colgroup class="ck-table_data-col" span="2"></colgroup>
  <colgroup class="ck-table_data-col" span="2"></colgroup>
  <thead>
    <tr>
      <td rowspan="2"></td>
      <th colspan="2" scope="colgroup">Mars</th>
      <th colspan="2" scope="colgroup">Venus</th>
    </tr>
    <tr>
      <th scope="col">Produced</th>
      <th scope="col">Sold</th>
      <th scope="col">Produced</th>
      <th scope="col">Sold</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <th scope="row">Teddy Bears</th>
      <td>50,000</td>
      <td>30,000</td>
      <td>100,000</td>
      <td>80,000</td>
    </tr>
    <tr>
      <th scope="row">Board Games</th>
      <td>10,000</td>
      <td>5,000</td>
      <td>12,000</td>
      <td>9,000</td>
    </tr>
  </tbody>
</table>
```

**Comment by baerkins** [comment]
Any update on the status of this? Or another way for a content author to easily add the scope attribute?

Thanks!
